config/locales/en/summary.yml

```diff
@@ -281,6 +281,14 @@ en:
         question: How much rent does your client pay, after taking away Housing Benefit?
         answers:
           description: "%{amount} every %{frequency}"
+      pays_council_tax:
+        question: Does your client pay Council Tax where they usually live?
+        answers:
+          <<: *YESNO
+      council_tax:
+        question: How much do they pay yearly?
+        answers:
+          description: "£%{amount}"
```

Comment:
Your money answer partial is handling the adding of the '£' - so ending up with '££' atm

Reply:
Ah merge issue well spotted